explanation/service-endpoint-encryption.rst

```diff
@@ -10,8 +10,7 @@ OpenStack supports enabling TLS via the Traefik application, which is the
 ingress point for all service endpoints.
 
 .. note::
-   Currently, only the TLS CA feature method is supported. This feature only
-   works with certificates signed by an external Certificate Authority.
+   Currently, only the TLS CA and TLS Vault feature methods are supported. TLS CA only works with certificates signed by an external Certificate Authority.
```

Comment:
It's good practice to spell out abbreviations in full in their first instance on the page, followed by the abbreviation in parentheses. I see that this isn't the first instance, but just thought I should comment on that since it's not the case in this page.

Reply:
I kept the same way the document was previously writen. I will add a `Certificate Authority` in paranthesis after the first TLS CA, to make it easier to understand the abreviation